db/migrate/20220817103152_add_reference_to_devices_employee.rb

```diff
@@ -0,0 +1,5 @@
+class AddReferenceToDevicesEmployee < ActiveRecord::Migration[6.1]
```

Comment:
@MadhaviBellam remove it for now as employees schema still not available. add this migration once pavan will merge his changes